examples/monitor-multiple-cluster-tls/cert-manager/ns1/cluster-client-secret.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: cert-manager.io/v1alpha2
```

Comment:
Can we just add some templates in `examples/cert-manager` with the TidbCluster as a special name like TIDB_CLUSTER_NAME in the templates, write a simple shell to generate the desired yamls from the templates? Then we no need to add so many yamls in this PR and also the ones in `examples/heterogeneous-tls/cert-manager`?

Reply:
And also consider the namespace, which will be more common to use.

Reply:
You're right. Ref issue: https://github.com/pingcap/tidb-operator/issues/3659